int32 id
int32 cls
float32 score
string lane_id
BoundingBox bbox
